docs/chromadb-compaction-strategy.md

```
@@ -0,0 +1,218 @@
+DynamoDB Stream → SQS FIFO → Lambda → Lock → Download Snapshot → Apply Updates → Upload Snapshot
+                                ↓
+                         (Lock contention)
+                         Multiple Lambdas fight for lock
+                         Losers fail fast, messages return to queue
+```
+
+### Key Components
+
+| Component | Configuration |
+|-----------|---------------|
+| Lambda Memory | 4096 MB |
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)